index.bs

```diff
@@ -390,6 +390,74 @@ by a custom configuration.
 {}
    ```
 
+# Security Considerations {#security-considerations}
+
+The Sanitizer API is intended to prevent DOM-Based Cross-Site Scripting
+by traversing a DocumentFragment and removing elements and attributes
+according to a configuration. The specified API must not support
+the construction of a Sanitizer object that leaves script-capable markup in
+and doing so would be a bug in the threat model.
+
+That being said, there are security issues which the correct usage of the
```

Comment:
I wonder if we can re-phrase that... IMHO, the distinguishing factor of whether we can protect users is whether it's a browser property (i.e., the browser knows exactly which elements/attributes contain script), or whether it's an application property (e.g. whether the app will rely on `id=` attributes in a certain fashion). In this first case, the Sanitizer can and will protect. In the second case, we'll gladly supply appropriate tooling, but we cannot know whether or how to apply it.

Reply:
I agree with what you say, but I don't think the application property / browser-property suggestion applies cleanly to this specific sentence. Especially given the following enumeration contains things like Server-Side XSS. Can you elaborate?

Reply:
My thinking is: There are clear, browser-defined script-y semantics. Those are the non-overridable defaults. Then there is everyhing else, which is not clear, browser-defined script-y semantics, like server-side XSS, or mXSS, or script gadgets, etc. In my mind, that's the dividing line between overridable and non-overridable.

I'll gladly let you decide on this, though.